cli/installer.rs

```diff
@@ -94,6 +105,7 @@ fn generate_executable_file(
   // TODO: remove conditionals in generated scripts
   let template = format!(
     r#"#!/bin/sh
```

Comment:
While you at it, could you remove those conditionals in generated scripts?

Reply:
Do you mean the branch of `$basedir/deno` and similar since deno binary have been moved to `.local`?

Reply:
Yeah, the "if" branches in both generated scripts - I don't really know what it was for but @ry wants it gone